<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="G1C2ML14286" name="com.mindtree.executeEngine.DriverScript" tests="5" failures="0" timestamp="24 Sep 2018 18:14:09 GMT" time="231.681" errors="1">
  <testcase name="testData" time="57.480" classname="com.mindtree.executeEngine.DriverScript"/>
  <testcase name="testData" time="41.632" classname="com.mindtree.executeEngine.DriverScript">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.mindtree.executeEngine.MapLocator.locate(MapLocator.java:20)
at com.mindtree.executeEngine.DriverScript.testData(DriverScript.java:186)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- testData -->
  <testcase name="testData" time="33.382" classname="com.mindtree.executeEngine.DriverScript"/>
  <testcase name="testData" time="54.593" classname="com.mindtree.executeEngine.DriverScript"/>
  <testcase name="testData" time="44.594" classname="com.mindtree.executeEngine.DriverScript"/>
</testsuite> <!-- com.mindtree.executeEngine.DriverScript -->
